.github/workflows/release-reuse.yml

```
@@ -0,0 +1,168 @@
+  id-token: write   # REQUIRED for Cosign keyless
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      image_ref: ${{ steps.meta.outputs.image_ref }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set image metadata
+        id: meta
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)